src/jsonapi-serializer/error.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+function ErrorDocument (errors) {
+  var jsonapi = {
+    errors: []
+  }
+
+  errors.forEach(function (error) {
+    var opts = {}
+
+    if (error.id) { opts.id = error.id }
+    if (error.status) { opts.status = error.status }
+    if (error.code) { opts.code = error.code }
+    if (error.title) { opts.title = error.title }
+    if (error.detail) { opts.detail = error.detail }
```

Comment:
Is this better or worse than

```jsx
['id', 'status', 'code', 'title', 'detail'].forEach((key) => {
  if (error[key]) { opts[key] = error[key] }
})
```

?

Reply:
I think it's worse.

Five simple `if` statements look pretty stupid, but even being braindead, I can understand the code and see it works. The loop is slightly more complicated even though it's still easy.

But the difference is small enough to be decided for personal preference and I like stupid and simple.